base/shell/explorer/tbsite.cpp

```diff
@@ -459,7 +459,7 @@ class CTrayBandSite :
         /* Enumerate all bands */
         while (SUCCEEDED(m_BandSite->EnumBands(uBand, &dwBandID)))
         {
-            if (SUCCEEDED(m_BandSite->GetBandObject(dwBandID, IID_PPV_ARG(IPersist, &pBand))))
+            if (dwBandID && SUCCEEDED(m_BandSite->GetBandObject(dwBandID, IID_PPV_ARG(IPersist, &pBand))))
```

Comment:
Similar code exists at other places. Is this the only place that needs such a check to be added? (Why?)
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=EnumBands

Reply:
hmm good concern. Why would EnumBands() function succeed but return a zero dwBandID?
(And also, why would a zero dwBandID be invalid? Does this means, absence of band? Why wouldn't the GetBandObject function instead handle the case of a dwBandID being zero ?)

Reply:
Those questions were raised after the current PRs state had 3 approvals, already. -> merge now.
This shouldn't stop us from trying to answer them. I will leave the conversation unresolved. Follow-up-PRs are allowed of course.